datasetinsights/estimators/faster_rcnn.py

```diff
@@ -263,30 +325,29 @@ def train_one_epoch(
     def evaluate(self, **kwargs):
         """evaluate given dataset."""
         config = self.config
-        test_dataset = create_dataset(config, TEST)
+        test_dataset = create_dataset(config, self.data_root, TEST)
         label_mappings = test_dataset.label_mappings
-        if self.config.system.dryrun:
-            test_dataset = create_dryrun_dataset(config, test_dataset, TEST)
-
```

Comment:
Why are you removing dry run? if we are not supporting dry run why we have create_dryrun_dataset method in this class?

Reply:
See above answer